### ISSUE TITLE
Design | Default Send/Checkboxes & Editing as a Subtask | Contact Center Guide Updates

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#63278]
## Background 

We need to update our contact center guides for the Email As a Channel/Default Send/Checkbox/editing as a subtask work to receive asynchronous feedback. 

According to [platform supports process guidance](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1686676305244/4fb520e97c488d1fd053d93c6497e09ab0880207?sender=uaa72d11015d3f7c704a64191), this should happen prior to staging review.

## Tasks 
- [x] update the google[ drive version of the document ](https://docs.google.com/document/d/1kp6jMQ5tU8UJ8XgKlcE-Srg9OedqaTTo/edit?usp=sharing&ouid=108263438832023933926&rtpof=true&sd=true)with the latest notification settings screens to include: editing as a subtask and the checkbox UI updates 
- [x] review w/PM and PO
- [ ] update the [contact center touchpoint ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63198)w/the latest version of the guide

## Acceptance Criteria 
- [x] notification settings in the guide have been updated to include the new checkboxes 
- [x] the guide also reflects the editing as a subtask work
- [x] Guide has been reviewed with PM

**Comment by mtcA6** [comment]
Section Overview: Notification Settings	75
User Access	75
Navigation	75
Functionality	76
Major Issues and Error Messages	81

**Comment by mtcA6** [comment]
@andaleliz keyboard shortcuts betrayed me here, so the section we need to update are in the prior comment, I trust your judgement on which screenshots you update, I already started today to update the copy. 

It's the notification settings part of this document, which also has the legacy flow we used for the editing as a subtask process. 
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/58511fa2-67a9-4604-a609-6299bc4cfcd3)

**Comment by andaleliz** [comment]
@mtcA6 I've added all the screenshots, and left a couple of comments/edits in suggesting mode. I think once those are squared away we can bring in Samara for her review.

A couple of comments on the other tasks here:
- I think the touchpoint ticket needs to be updated to remove references to email as a channel. Also, I noticed the account security section is outdated, so we should probably include those updates in this round of reviews as well.  I created a ticket to update the screenshots (#63939) so other updates are ready for you to rock 'em.
- One of my tasks is to update the version guide. Should it be 5.4, or is 5.3 the update? I see the date has been updated so unclear about the version number.
- The AC of the "guide has been submitted" - can I remove that as a definition of done for the design side of things? I believe that's something you/Samara will tackle.

**Comment by mtcA6** [comment]
When you say touchpoint ticket what are you referring to? The design intent ticket or something else? 

I was gonna figure out the numbering before we submit it. Yea Samara said she wasn't sure what was up w the numbering. 

Yep that's fine to change the AC! I understand the boundary now for updating the guide.

**Comment by andaleliz** [comment]
@mtcA6  Thank you for all the info!

>When you say touchpoint ticket what are you referring to? The design intent ticket or something else?

The ticket you referenced in the description

**Comment by andaleliz** [comment]
@mtcA6 are you going to address the rest of the items needed to get this submitted? Trying to figure out if I need to leave this open.  Thanks!

**Comment by mtcA6** [comment]
yep, @andaleliz had the guide open today and was looking at it and kept getting pulled away into other things. reassigned the ticket to me I'll close it out when I upload it against the collab ticket. 

I may open another ticket for other sections of the guide for help grabbing screenshots but thanks for your help for now.

**Comment by andaleliz** [comment]
@mtcA6 awesome, thank you! Yep, that works for me if you want to open another ticket, or re-assign this back to me for further work.
